ironfish/src/consensus/consensus.ts

```diff
@@ -43,9 +43,13 @@ export const MAX_SYNCED_AGE_MS = 60 * 1000
 export const MAX_REQUESTED_BLOCKS = 50
 
 /**
- * The network has a target block time of 15 seconds
+ * The average time that all blocks should be mined
  *
  * NOTE: This is not used in target calculation, or IRON_FISH_YEAR_IN_BLOCKS.
- * Ideally, it should be in a future refactor.
  */
-export const TARGET_BLOCK_TIME_MS = 15 * 1000
+export const TARGET_BLOCK_TIME_IN_SECONDS = 60
+
+/**
+ * The time range when difficulty and target not change
+ */
+export const FREEZE_TIME_IN_SECONDS = 10
```

Comment:
since we're using this to determine our bucket size, maybe something like BUCKET_THRESHOLD_IN_SECONDS? idk just trying to figure out how to make this clearer